src/Toast/index.tsx

```diff
@@ -14,16 +13,40 @@ import IconButton from '../IconButton';
 export const TOAST_CLOSE_LABEL_TEXT = 'Close';
 export const TOAST_DELAY = 5000;
 
+interface ToastAction {
+  label: string;
+  href?: string;
+  onClick?: () => void;
```

Comment:
[curious] Is the `Event` associated with the click passed to this `onClick` as an arg? If so, should it be included in the type?

Reply:
Updated, thanks (https://github.com/openedx/paragon/pull/3434/commits/4944236d315e5a9829654db6af0d8f58783d4394)